Firebase/Database/Persistence/FLevelDBStorageEngine.m

```diff
@@ -878,7 +878,17 @@ - (id)fixDoubleParsing:(id)value
     // values, including 2.47. Because we use the exact bytes for hashing on the
     // server this will lead to hash mismatches. The parser of NSNumber seems to
     // be more in line with what the server expects, so we use that here
```

Comment:
Can you move this existing comment into the `else if if ([value isKindOfClass:[NSNumber class]])` block?

Reply:
Certainly, but the essential idea of what is going on inside both the NSDecimalNumber case and the NSNumber case is the same, so does this comment not apply to both?

I am updating the PR with the moved comment, but I think that perhaps it belongs to the entire method.